css/style.css

```diff
@@ -72,4 +73,14 @@ li {
 }
 span {
     color: black;
-}
\ No newline at end of file
+}
+@media screen and (max-width: 768px){
+    .header {font-size: 18px;
+        flex-direction: column;}
+    .introduction {font-size: 20px;}
+    .skills {font-size: 20px;}
+    .education {font-size: 20px;}
+    .avatar-container {width: 100%;}
+    .personal-information {width: 100%;}
+    .main-container{flex-direction: column;}
```

Comment:
Please, format css in this way: 
.header {
  font-size: 18px;
  flex-direction: column;
}

Reply:
fixed